src/lib/sql/sql_translator.cpp

```diff
@@ -136,19 +136,10 @@ SQLTranslator::SQLTranslator(const UseMvcc use_mvcc)
     : SQLTranslator(use_mvcc, nullptr, std::make_shared<ParameterIDAllocator>(),
                     std::unordered_map<std::string, std::shared_ptr<LQPView>>{}) {}
 
-std::vector<ParameterID> SQLTranslator::parameter_ids_of_value_placeholders() const {
-  const auto& parameter_ids_of_value_placeholders = _parameter_id_allocator->value_placeholders();
-  auto parameter_ids = std::vector<ParameterID>{parameter_ids_of_value_placeholders.size()};
-
-  for (const auto& [value_placeholder_id, parameter_id] : parameter_ids_of_value_placeholders) {
-    parameter_ids[value_placeholder_id] = parameter_id;
-  }
-
-  return parameter_ids;
-}
-
-std::vector<std::shared_ptr<AbstractLQPNode>> SQLTranslator::translate_parser_result(
+std::pair<std::vector<std::shared_ptr<AbstractLQPNode>>, TranslationInfo> SQLTranslator::translate_parser_result(
```

Comment:
Change this to `SQLTranslationResult` and make the vector a part of it? That would save us from having to remember what `.first` and `.second` are.